wave-utils/src/test/groovy/io/seqera/wave/util/DockerHelperTest.groovy

```diff
@@ -510,4 +510,27 @@ class DockerHelperTest extends Specification {
                     apt-get update -y && apt-get install -y procps
                 '''.stripIndent()
     }
+
+    def 'should create singularity content from spack file'(){
+        given:
+        def SPACK_OPTS = [ commands:['USER hola']]
```

Comment:
Not really related to this PR that everything seems correct, but @pditommaso I've some doubts about this `commands` parameter. 

Where this comes from? Is the user that can add custom commands to the recipe? then, unless we add the prefix `RUN` when building the `Dockerfile`, these commands are not container engine agnostic and they should be different if they're using Singularity or Docker.

Reply:
Do we want to make them engine agnostic? or is that ok?

Reply:
the `commands` is the setting provided by the user via the option `wave.build.spack.command`. It's not expected to be portable across different container engines 

https://www.nextflow.io/docs/edge/wave.html#advanced-settings